lib/datastore/transaction.js

```diff
@@ -143,10 +141,10 @@ Transaction.prototype.rollback = function(callback) {
  * Commit the remote transaction and finalize the current transaction instance.
  *
  * @param {function} callback - The callback function.
+ * @private
```

Comment:
I think we did it to be Node convention-friendly. Async functions usually gets a function to execute when you're done.